junit-platform-engine/src/test/java/io/cucumber/junit/platform/engine/DiscoverySelectorResolverTest.java

```diff
@@ -51,25 +52,19 @@
 import static org.junit.platform.engine.discovery.DiscoverySelectors.selectUniqueId;
 import static org.junit.platform.engine.discovery.DiscoverySelectors.selectUri;
 
+@WithLogRecordListener
```

Comment:
Unfortunately due to the architecture we can't share test code between modules in this project. I think that duplicating the class would not be the least disturbing solution. While it duplicated the code, it preserves the modular nature of the project.